server/src/main/java/com/thoughtworks/go/server/database/H2Database.java

```diff
@@ -135,7 +135,7 @@ public BasicDataSource createDataSource() {
     }
 
     private String dburl(Boolean mvccEnabled) {
-        return "jdbc:h2:" + systemEnvironment.getDbPath() + "/" + configuration.getName()
+        return "jdbc:h2:" + systemEnvironment.getDbPath().getAbsolutePath() + "/" + configuration.getName()
```

Comment:
Was `getAbsolutePath()` deliberately added?

Reply:
Yes. The DB is now being updated during the config migration and it required absolute path for the same.